Dockerfile

```diff
@@ -45,10 +45,7 @@ RUN cd /app \
 ######################################################################
 # Node stage to deal with static asset construction
 ######################################################################
-FROM node:16 AS superset-node
-
-ARG NPM_VER=7
-RUN npm install -g npm@${NPM_VER}
+FROM node:16-slim AS superset-node
```

Comment:
There have been issues with recent versions of `npm@8`, so I'd propose sticking to `npm@7` for now

Reply:
@villebro As I remember there're no any breaking changes in `npm@8`
These lines were added when we used `NodeJS 12` in order to use `npm@7` instead of `npm@6`

Reply:
I remember `npm@8` ceasing to work when building `superset-frontend` after a certain patch version (I can't remember which). It may have been resolved since, but I'm not aware of it having been cleared up yet.

Reply:
**_Version 8 is identical to version 7 except for an engines update_**

Maybe a minor version of `npm@8` was broken but now latest `NodeJS 16` has stable `npm@8`. If we see github action all is OK and `npm@8` is stable

Reply:
@lyndsiWilliams @kgabryje @zhaoyongjie Should we use latest `npm@8` to install & build UI in docker container?

Reply:
I checked, and apparently the `npm@8` issue has resolved itself, so this should be ok.

Reply:
@villebro @EugeneTorap I am using `npm@8` now, it works for me.